c2rust-transpile/tests/snapshots/snapshots__transpile@macros.c.snap

```diff
@@ -370,7 +370,7 @@ pub static mut fns: fn_ptrs = {
     init
 };
 #[no_mangle]
-pub static mut p: *const fn_ptrs = unsafe { &fns as *const fn_ptrs };
```

Comment:
It's strange that this is unsafe, but that isn't changed by these commits. Just a note to self to look into this.

Reply:
Ah, I didn't realize that it was `static mut`.